kanidm_tools/src/cli/account.rs

```diff
@@ -281,6 +283,33 @@ impl AccountOpt {
                         }
                     }
                 }
+                AccountCredential::GenerateBackupCode(acsopt) => {
+                    let client = acsopt.copt.to_client();
+                    match client.idm_account_primary_credential_generate_backup_code(
+                        acsopt.aopts.account_id.as_str(),
+                    ) {
+                        Ok(s) => {
+                            println!("GenerateBackupCode success.");
+                            println!("{:#?}", s);
```

Comment:
I know @victorcwai said this was oky, but I think we should have this as `{}` rather than `{:#?}`. The former is "Display" the latter is "Debug". When we are debbuging, or an error, we want that. But for "user facing" information, the default should be to provide it with "display". There are some subtle examples of this distinction in the proto files, where we seperate "debug" and "display" based on how and when we are presenting the information.